src/actions/authActions.js

```diff
@@ -219,8 +218,7 @@ export const loginAction = (
         // if we disable feature flag or end beta testing program
         // while user has set PPN or BTC as active network
         const revertToDefaultNetwork =
-          (!smartWalletFeatureEnabled && blockchainNetwork === BLOCKCHAIN_NETWORK_TYPES.PILLAR_NETWORK) ||
-          (!bitcoinFeatureEnabled && blockchainNetwork === BLOCKCHAIN_NETWORK_TYPES.BITCOIN);
+          !smartWalletFeatureEnabled && blockchainNetwork === BLOCKCHAIN_NETWORK_TYPES.PILLAR_NETWORK;
```

Comment:
I guess this will conflict with latest develop changes that include Smart Wallet feature flag removal, so I guess you can just remove whole revert stuff.